src/Nerve_GIC/utilities/KeplerMapperVisuFromTxtFile.py

```diff
@@ -1,9 +1,7 @@
 #!/usr/bin/env python
 
-import km
-import numpy as np
-from collections import defaultdict
 import argparse
+from gudhi.cover_complex import save_to_html_
```

Comment:
Also need to change it in this file?

Reply:
Oops indeed I forgot this one, good catch!